examples/partitionContainer/README.md

```diff
@@ -0,0 +1,68 @@
+# Partitioning Container Example for Iguana
+
+## What does it do?
+
+The container will partition the hard disk into a user specified number of partitions. The user can define the specific size of each partition and what type of partition it is _(eg. EFI, swap, root etc...)_. Users also have the option of specifying each partition's type through UUID (https://uapi-group.org/specifications/specs/discoverable_partitions_specification/) as well as its mount point. 
+## Inputting Configurations
+All of the configuration will be passed through a JSON file with the following format:
+```json
+    {
+        "general": {
+            "label": "gpt",
+            "initial_gap": "2MB"
+        },
+        "devices": {
+            "device_name": {
+                "blkSize": "512B",
+                "partitions": [{
+                        "size": "1000MiB",
+                        "type": "efi"
+                    },{
+                        "size": "2000MiB",
+                        "type": "linux",
+                        "optional": {
+                            "UUID": "75250d76-8cc6-458e-bd66-bd47cc81a812",
+                            "mountPoint": "/usr/"
+                        }    
+                    }
+                ]
+            }
+        }
+    }
```

Comment:
As an example we should add some real world usable config.
I suggest:
efi - 512MiB
swap - 2GiB
root - 16GiB